source/includes/quick-start/query-output.rst

```diff
@@ -0,0 +1,25 @@
+When you run ``main.go``, it should output the details of the
+movie from the sample dataset which will look something like this:
+
+.. code-block:: json
+
+   [
```

Comment:
This is related to my comment about using `bson.D` in the `main.go` code. Because of the reasons explained there, this JSON is an array and doesn't actually reflect the document that's stored in the DB. When switching to `bson.M` in the code, please also update this file with the new JSON.